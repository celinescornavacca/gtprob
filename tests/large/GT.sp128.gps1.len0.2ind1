(((((((EA0,FA0),(NB0,OB0)),(CA0,(((VC0,WC0),NA0),((GA0,((WA0,XA0),(MC0,NC0))),(((LA0,MA0),IA0),(AC0,BC0)))))),(ZA0,(IE0,JE0))),(((((TA0,(IB0,(((ED0,FD0),CD0),(WD0,(SE0,TE0))))),(RB0,(VB0,WB0))),(((((LD0,MD0),DD0),(ID0,JD0)),(((KE0,LE0),DE0),(ZB0,(EC0,FC0)))),((PB0,QB0),AB0))),(PA0,QA0)),((SA0,(YA0,(JC0,KC0))),(JB0,(WE0,XE0))))),((((GE0,HE0),(AD0,BD0)),SB0),(HA0,(YC0,ZC0)))),((GB0,HB0),((BA0,(((DA0,((XD0,(OE0,PE0)),KA0)),((RC0,SC0),((GC0,HC0),CC0))),((((AE0,BE0),XC0),((QE0,RE0),UA0)),((MB0,((UD0,VD0),OC0)),((TB0,UB0),(EE0,FE0)))))),(((((ND0,(YD0,ZD0)),(((((CB0,(KB0,LB0)),((DB0,EB0),BB0)),((XB0,YB0),VA0)),((((FB0,((ME0,NE0),((SD0,TD0),KD0))),(((UE0,VE0),CE0),(TC0,UC0))),RA0),OA0)),(QD0,RD0))),AA0),(JA0,(GD0,HD0))),((IC0,(PC0,QC0)),((LC0,(OD0,PD0)),DC0))))));
