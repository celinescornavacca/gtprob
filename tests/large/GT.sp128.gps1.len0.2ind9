(((((HB0,IB0),MA0),((UB0,(FE0,GE0)),WA0)),BA0),((((((((KA0,((JC0,KC0),(QE0,RE0))),(PA0,(NC0,OC0))),((WE0,XE0),((EB0,FB0),(SA0,((IE0,JE0),HE0))))),(((DA0,(((IA0,JA0),(QA0,(ME0,NE0))),(((UA0,VA0),((YB0,(ZB0,(HD0,(OE0,PE0)))),TA0)),((OA0,((XB0,((BD0,CD0),ZC0)),DB0)),(ZD0,AE0))))),((((UC0,VC0),AC0),((TD0,UD0),KB0)),(BB0,CB0))),(((((NB0,OB0),MB0),JB0),EA0),((PC0,QC0),(KE0,LE0))))),(((((((EC0,FC0),LB0),GB0),XA0),HA0),(XD0,YD0)),(RA0,(BE0,CE0)))),((((((FD0,GD0),(XC0,YC0)),((RD0,SD0),RC0)),LA0),(((KD0,LD0),RB0),(FA0,GA0))),((((TB0,(VB0,WB0)),(PB0,QB0)),((HC0,IC0),NA0)),(((((SC0,TC0),(BC0,CC0)),(DE0,EE0)),((GC0,((DD0,ED0),WC0)),YA0)),((SE0,TE0),OD0))))),(((MD0,ND0),((ID0,JD0),SB0)),((((ZA0,AB0),(AD0,(VD0,WD0))),(UE0,VE0)),(CA0,((DC0,(LC0,MC0)),(PD0,QD0)))))),AA0));
