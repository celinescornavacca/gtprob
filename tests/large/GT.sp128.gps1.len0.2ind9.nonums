(((((HB,IB),MA),((UB,(FE,GE)),WA)),BA),((((((((KA,((JC,KC),(QE,RE))),(PA,(NC,OC))),((WE,XE),((EB,FB),(SA,((IE,JE),HE))))),(((DA,(((IA,JA),(QA,(ME,NE))),(((UA,VA),((YB,(ZB,(HD,(OE,PE)))),TA)),((OA,((XB,((BD,CD),ZC)),DB)),(ZD,AE))))),((((UC,VC),AC),((TD,UD),KB)),(BB,CB))),(((((NB,OB),MB),JB),EA),((PC,QC),(KE,LE))))),(((((((EC,FC),LB),GB),XA),HA),(XD,YD)),(RA,(BE,CE)))),((((((FD,GD),(XC,YC)),((RD,SD),RC)),LA),(((KD,LD),RB),(FA,GA))),((((TB,(VB,WB)),(PB,QB)),((HC,IC),NA)),(((((SC,TC),(BC,CC)),(DE,EE)),((GC,((DD,ED),WC)),YA)),((SE,TE),OD))))),(((MD,ND),((ID,JD),SB)),((((ZA,AB),(AD,(VD,WD))),(UE,VE)),(CA,((DC,(LC,MC)),(PD,QD)))))),AA));
