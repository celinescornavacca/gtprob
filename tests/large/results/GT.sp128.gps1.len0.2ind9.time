3.72user 0.13system 0:04.04elapsed 95%CPU (0avgtext+0avgdata 63408maxresident)k
144inputs+0outputs (0major+33912minor)pagefaults 0swaps
