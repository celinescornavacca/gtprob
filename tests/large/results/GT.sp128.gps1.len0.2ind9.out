-166.184102838
